dashboard/app/views/pd/regional_partner_contact/thanks.html.haml

```diff
@@ -1,17 +1,23 @@
 - if @regional_partner
   Thank you for expressing your interest in bringing Computer Science to your school or
-  district.  Your Code.org regional partner is
+  district.  Your Code.org Regional Partner is
   = "#{@regional_partner.name}."
   A representative from
   = @regional_partner.name
   will receive your contact information and contact you soon.
+
+  In the meantime, take a look
+  = link_to 'here for the K-12 Code.org curriculum options', courses_path
+  available for no cost to all schools
 - else
   %p
     Thank you for expressing your interest in bringing computer science to your school or
-    district. There is not currently a Code.org regional partner in your area. The Code.org
+    district. There is not currently a Code.org Regional Partner in your area. The Code.org
     team will receive your interest form and forward it to a regional partner in your
     community if and when they become available.
 
   In the meanwhile, take a look
```

Comment:
nit: meantime on line 9, meanwhile here - should we pick a word and be consistent?